module1/src/com/epam/branching/HomeTask_1_2_1.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.branching;
+
+public class HomeTask_1_2_1 {
+    public static void main(String[] args) {
+        int a = 90;
+        int b = 45;
+        int c ;
+        if ( ( a + b) < 180 ) {
+            c = 180 - (a + b);
+            System.out.println("Треугольник с такими углами существует.");
+
+            if (a == 90 || b == 90 || c ==90){
+                System.out.println("И он являеться прямоугольным");
```

Comment:
1)You don't need variable c
2)you can count sum of a+b and assign to variable cause you use it twice
3)let's still select the constants
4)let's use double for angles
```suggestion
    private static final double SUM_OF_ANGLES = 180.0;
    private static final double RIGHT_ANGLE = 90.0;
    
    public static void main(String[] args) {
        double a = 90;
        double  b = 45;
        double sumOfTwoAngles = a + b;
        if (sumOfTwoAngles < SUM_OF_ANGLES) {
            if (a == RIGHT_ANGLE || b == RIGHT_ANGLE || sumOfTwoAngles == RIGHT_ANGLE) {
                System.out.println ("This is a right-angled triangle");
            } else {
                System.out.println("This isn't a right-angled triangle");
            }
        } else {
            System.out.println ("A triangle with defined angles doesn't exist!");
        }
    }
```